kolibri/deployment/default/urls.py

```diff
@@ -24,7 +24,7 @@
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^$', include('kolibri.core.urls')),
+    url(r'^kolibri/', include('kolibri.core.urls')),
     url(r'', include('kolibri.content.urls')),
     url(r'^api/', include('kolibri.auth.urls')),
     url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
```

Comment:
By defining all these apps' url configurations in a project environment, we loose control, since essentially it's an external environment. Should really avoid this @rtibbles @jamalex @MCGallaspy @indirectlylit -- let's refactor it to live in kolibri.core.urls ? (not related to this PR, but slightly)